assignments/callbacks.js

```diff
@@ -27,29 +27,57 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
 
+getLength(items, (length) => {
+  console.log(length);
+})
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr.pop());
```

Comment:
While `pop()` is useful for grabbing the last element, it also has the side effect of mutating the original array, which sometimes isn't what we want. Passing in `arr[arr.length - 1]` would grab the last item without mutating the array.